docker-compose.yml

```diff
@@ -873,10 +875,12 @@ services:
               metadataServerUrls="http://kafka1:8091,http://kafka2:8092";
       KAFKA_REST_SSL_CIPHER_SUITES: "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256"
 
-      # RBAC and MDS
+      # Enable OAuth for REST Proxy's embedded Kafka client that accesses and manages consumer groups and topics
```

Comment:
One alternative phrasing to consider:

"Enable bearer token authentication, which allows the identity of the end user to be propagated to Kafka for authorizing access to Kafka topics and consumer groups"

"Enable OAuth for REST Proxy's embedded Kafka client that accesses and manages consumer groups and topics" sounds slightly more applicable to the JAAS config that precedes.

My wording probably isn't better, so please feel free to ignore.